build/terraform/gke/module.tf

```diff
@@ -97,17 +97,21 @@ variable "feature_gates" {
   default = ""
 }
 
+variable "kubernetes_version" {
```

Comment:
Variable was used here so that we can make changes to this file more rare. And imagine the situation when between Agones releases Kubernetes version would be deprecated but with Terraform we are cloning the Agones repo (as in docs) only at release points. Another thing is if we want to compare Terraform configs of Agones at version `1.6.0` and `1.7.0` to perform regression testing for example, then we can select the same Kubernetes version to compare at the same environment.